sites/platform/src/increase-observability/metrics/dedicated-generation-3.md

```diff
@@ -26,42 +26,42 @@ This example should give you an idea of how the metrics appear.
 {{% names/dedicated-gen-3 %}} environments metrics show resource usage for each app, service, and worker container
 across all hosts.
 
-This reference project has a single app, two services (MySQL and Redis), and one worker.
+This reference project has a single app, two services (Postgresql and Redis), and one worker.
 
 Once you've read the metrics, see [recommendations for action](./_index.md#dedicated-gen-3-environments).
 
 ### App container
 
 Metrics graphs for the app container show CPU, RAM, and disk allocation and usage across all hosts.
 The persistent disk has been configured in the [app configuration](/create-apps/app-reference/single-runtime-image.md#top-level-properties)
-at 4.86&nbsp;GB, while the temporary disk is 3.99&nbsp;GB by default.
+at 9.58&nbsp;GB, while the temporary disk is 49.04&nbsp;GB.
 
-![All of the metrics for the app container](/images/metrics/app-container-gen3.png)
+![All of the metrics for the app container](/images/metrics/DG3-app-container.png)
 
 ### Service containers
 
 Metrics graphs for the service containers show CPU, RAM, and disk allocation and usage across all hosts.
 
-#### MySQL
+#### Postgresql
 
-Metrics graphs for the MySQL service container show CPU, RAM, and disk allocation and usage across all hosts.
+Metrics graphs for the Postgresql service container show CPU, RAM, and disk allocation and usage across all hosts.
```

Comment:
`PostgreSQL`

Reply:
or maybe, as it is the identifier of the service: 
`postgresql` (as snippet)

Reply:
but needs to be coherent in each paragraph, as they all start with: 
```
Metrics graphs for the <service_name> container...
```
--> `redis`